config/service_qos.go

```diff
@@ -37,81 +39,104 @@ func init() {
 	for k, v := range shannonQoSTypes {
 		ServiceQoSTypes[k] = v
 	}
+	for k := range shannonEVMChainIDs {
+		ServiceQoSTypes[k] = ServiceIDEVM
+	}
+
 	for k, v := range morseQoSTypes {
 		ServiceQoSTypes[k] = v
 	}
-	for k, v := range shannonTestQoSTypes {
-		ServiceQoSTypes[k] = v
+	for k := range morseEVMChainIDs {
+		ServiceQoSTypes[k] = ServiceIDEVM
 	}
 }
 
+// GetEVMChainID returns the hexadecimal EVM chain ID for a given service ID.
```

Comment:
Doesn't have to be hex in Shannon.

```suggestion
// GetEVMChainID returns the EVM chain ID for a given service ID.
```